### DISCUSSION TITLE
[1.7.5067] GitHub Copilot could not connect to server

### DISCUSSION THREAD

**Discussion by sv3nnie** [community/community#11787] [Copilot Conversations] [answered]
Hi,

In version 1.7.5067 (released 2 days ago) I am unable to use Copilot because of the following error: 
![image](https://user-images.githubusercontent.com/67653224/154822062-286c96e2-b4c1-4e94-b610-e8e5f002c1ec.png)

I downgraded back to 1.7.5003 which works perfectly fine for me.

**Comment by davecheney** [comment]
It looks like something is interfering with your https traffic. Is it possible that your employer, internet service provider, or government may be interfering with your SSL traffic?

**Reply by sv3nnie** [reply]
The domain was indeed blocked due to ''Ads or Trackers'' . What exactly does this domain do and why was it added in version 1.7.5067?

**Reply by davecheney** [reply]
I’ve raised a bug with the vscode extension developers.

**Reply by Arshad-ullah** [reply]
Same issue

**Reply by Arshad-ullah** [reply]
How to solve this issue please?

**Comment by hsenag** [comment, marked as answer]
Sorry for the problems here. If you're having this problem, could you try again with the latest version?
